packages/common/src/schemas/index.ts

```diff
@@ -47,7 +47,7 @@ const trackMetadataSchema = {
 
 export const newTrackMetadata = (fields, validate = false) => {
   const validFields = validate
-    ? pick(fields, Object.keys(trackMetadataSchema).concat(['track_id']))
+    ? pick(fields, Object.keys(trackMetadataSchema).concat(['track_id', 'track_cid']))
```

Comment:
nice, would we want to just add this to trackMetadataSchema?

Reply:
that's what im wondering too